.packit.yaml

```diff
@@ -38,3 +38,54 @@ jobs:
       # do not get the version from a tag (git describe) but from the spec file
       get-current-version:
         - grep -oP '^Version:\s+\K\S+' packaging/command-line-assistant.spec
+
+  # Test jobs
```

Comment:
Please consider using anchor and referencing that in the "e2e" only modifying what we need to.
On one hand, it makes the config shorter, on the other hand it loses a bit of a readability IMO.

Reply:
+1

Reply:
Done